pkg/kubectl/bind-apiservice/plugin/servicebindingrequest.go

```diff
@@ -103,8 +107,10 @@ func (b *BindAPIServiceOptions) createAPIServiceBindings(ctx context.Context, co
 				}
 
 				if existing.Spec.KubeconfigSecretRef.Namespace == "kube-bind" && existing.Spec.KubeconfigSecretRef.Name == secretName {
+					fmt.Fprintf(b.Options.IOStreams.ErrOut, "Updating kubeconfig in secret kube-bind/%s for existing APIServiceBinding %s.\n", secretName, existing.Name) // nolint: errcheck
 					_, err = b.ensureKubeconfigSecretWithLogging(ctx, kubeconfig, secretName, kubeClient)
-					return nil, err
+					bindings = append(bindings, existing)
+					continue nextRequest
```

Comment:
the label is so many lines above 😱

Reply:
this function needs a refactor AND a unit test.